test/strings/basic.jl

```diff
@@ -240,6 +240,9 @@ end
     @test last(eachindex("foobar")) === lastindex("foobar")
     @test done(eachindex("foobar"),7)
     @test eltype(Base.EachStringIndex) == Int
+        == eltype(Base.EachStringIndex{String})
```

Comment:
According to `git grep`, there's no other occurrence of `==` at the beginning of a line in the codebase. Otherwise LGTM!

Reply:
Yes, the `==` needs to go at the end of the previous line or we'll stop parsing operators.